packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -17,10 +18,12 @@ describe('<TablePagination />', () => {
   before(() => {
     shallow = createShallow({ dive: true });
     mount = createMount();
+    consoleErrorMock.spy();
```

Comment:
Could we scope the console spy to the suite that tests the warnings? I would help to debug the other tests. The console output can be useful.